crates/docs/src/lib.rs

```diff
@@ -163,14 +187,47 @@ pub fn generate_docs_html(root_file: PathBuf) {
     println!("🎉 Docs generated in {}", build_dir.display());
 }
 
-fn sidebar_link_url(module_name: &str) -> String {
+fn module_link_url(module_name: &str) -> String {
     format!("{}{}", base_url(), module_name)
 }
 
 fn page_title(package_name: &str, module_name: &str) -> String {
     format!("<title>{module_name} - {package_name}</title>")
 }
 
+fn render_package_index(root_module: &LoadedModule) -> String {
```

Comment:
I'm not sure whether this is efficient. I tried `render_package_index<'a, I: Iterator<Item = &'a ModuleDocumentation>>(modules: I) -> String {...}` and `render_package_index(modules: &VecMap<ModuleId, ModuleDocumentation>) -> String {...}`. Each gave me different problems.

I figured the experienced Rustaceans here might be able to help!

Reply:
This is easy to read and thus maintain, I don't think this needs any further optimization :+1: